### PR TITLE
Display arbitrary NFTs by defining a Nacho NFT collection -- dev only

### PR DESCRIPTION
**Pull request by paninaro**
The NFT gallery selection dropdown will display a "Nacho NFTs" option if the local storage pref "nachoNFTs" is defined. The pref value is a string of a comma-separated NFT identifiers.

e.g.

"nft1sy37ezgaqjzg3mg3pwhltvz8ukc3uh9yaeagrs46qj4l8mdy7pmsun32tp, nft1k9c4cwmyctwzf57xzfaaghm20erdz23wzz3jsg0uwyft7qx5eeeqq4n93s, nft1ks2cqah7u6hjtj2q8vaem9a8e9n6v8rtxlzxzjs6zev8x3djevtq2kd8wn"

This is useful for debugging NFTs that are owned by another party.

### CODE REVIEW

**Comment by paninaro** [comment]
image

**Comment by hoffmang9** [comment]
🤩

File: packages/api-react/src/services/wallet.ts [hunk 1]
```diff
@@ -2012,6 +2048,11 @@ export const walletApi = apiWithTag.injectEndpoints({
           service: NFT,
           endpoint: () => walletApi.endpoints.getNFTs,
         },
+        {
+          command: 'onNFTCoinAdded',
+          service: NFT,
+          endpoint: () => walletApi.endpoints.getWallets,
+        },
```
Comment (paninaro) [inline packages/api-react/src/services/wallet.ts:2055]: This was for debugging purposes. Will remove.

**Review by seeden — APPROVED** [review]
lgtm

File: packages/gui/src/components/nfts/NFTProfileDropdown.tsx [hunk 2]
```diff
@@ -175,69 +175,32 @@ export default function NFTProfileDropdown(props: NFTGallerySidebarProps) {
...
+            <PermIdentityIcon />
+          </ListItemIcon>
+          {profile.name}
+        </MenuItem>
+      ))}
+      {haveNachoNFTs && (
+        <MenuItem
+          key="nacho"
+          onClick={() => handleWalletChange(-1)}
+          selected={walletId === -1}
```
Comment (seeden) [inline packages/gui/src/components/nfts/NFTProfileDropdown.tsx:197]: please add close attribute

Comment (paninaro) [reply packages/gui/src/components/nfts/NFTProfileDropdown.tsx:196]: Fixed in 84e7f5afab0b7e6c9f0c3dbba34edd413ac41912

**Review by seeden — APPROVED** [review]
lgtm

**Pull request merged**